docs/webdriver-bidi.md

```diff
@@ -28,24 +24,79 @@ Below is an example of launching Firefox or Chrome with WebDriver BiDi:
 ```ts
 import puppeteer from 'puppeteer';
 
-const browser = await puppeteer.launch({
-  product: 'firefox', // or 'chrome'
-  protocol: 'webDriverBiDi',
+const firefoxBrowser = await puppeteer.launch({
+  product: 'firefox', // WebDriver BiDi is used by default.
+});
+const page = await firefoxBrowser.newPage();
+...
+await firefoxBrowser.close();
+
+const chromeBrowser = await puppeteer.launch({
+  product: 'chrome',
+  protocol: 'webDriverBiDi'
```

Comment:
Nit: can we add `// CDP would be used by default.` for people who directly skip to the code snippet and did not read the intro?